tools/pd-ctl/tests/scheduler/scheduler_test.go

```diff
@@ -621,6 +621,7 @@ func (suite *schedulerTestSuite) checkHotRegionSchedulerConfig(cluster *pdTests.
 		"min-hot-key-rate":        float64(10),
 		"min-hot-query-rate":      float64(10),
 		"src-tolerance-ratio":     1.05,
+		"disabled":                false,
```

Comment:
Is it allowed to be set through `scheduler config xxx`?

Reply:
In current pr, it is allowed. Do we need mask it for users?

Reply:
I prefer to make it only for internal use purpose.

Reply:
https://github.com/tikv/pd/pull/8567/commits/2765b8ce679d5ea6bec11bebafe1c850ff74a750

https://github.com/tikv/pd/pull/8567/commits/590f2c1486c6d909efa5eaf53032dc623eed024c

done